php/WP_CLI/Runner.php

```diff
@@ -326,8 +326,17 @@ public function run_command( $args, $assoc_args = array(), $options = array() )
 		}
 	}
 
-	private function _run_command() {
+	private function _run_command( $exiting = true ) {
```

Comment:
I'll remove the arg and rename the method `_run_command_and_exit()` and have the case after the 1st help attempt just use `run_command( args...)`.